ext/lib/crypto/mbedtls/Kconfig

```diff
@@ -59,3 +59,24 @@ config MBEDTLS_INSTALL_PATH
 	This option holds the path where the mbedTLS libraries and headers are
 	installed. Make sure this option is properly set when MBEDTLS_LIBRARY
 	is enabled otherwise the build will fail.
+
+config MBEDTLS_ENABLE_HEAP
+	bool "Enable global heap for mbed TLS"
+	default n
+	depends on MBEDTLS
+	help
+	This option enables the mbedtls to use the heap. This setting must
+	be global so that various applications and libraries in Zephyr do not
+	try to do this themselves as there can be only one heap defined
+	in mbedtls. If this is enabled, then the Zephyr will, during the device
+	startup, initialize the heap automatically.
+
+config MBEDTLS_HEAP_SIZE
+	int "Heap size for mbed TLS"
+	default 15000
```

Comment:
That's super high, for most basic AES operation it's more like below 1Kb. High value is usually necessary for tls process, which needs to store keys etc...

You could set a small value by default, and tell in the help that depending on the usage of mtls it might need a large raise.